internal/mempool/v0/clist_mempool_test.go

```diff
@@ -121,11 +121,11 @@ func TestReapMaxBytesMaxGas(t *testing.T) {
 		{20, 0, -1, 0},
 		{20, 0, 10, 0},
 		{20, 10, 10, 0},
-		{20, 24, 10, 1},
+		{20, 28, 10, 1}, // account for overhead in Data{}
 		{20, 240, 5, 5},
-		{20, 240, -1, 10},
-		{20, 240, 10, 10},
-		{20, 240, 15, 10},
+		{20, 280, -1, 10},
+		{20, 280, 10, 10},
+		{20, 280, 15, 10},
```

Comment:
this is due to the increased size of proto `Data`

see this func https://github.com/celestiaorg/celestia-core/blob/8ea76d0823d56b68504b80fa394cfa6f39e5dfce/types/tx.go#L154-L160